index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <script src="https://kit.fontawesome.com/d55257b723.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header">
+        <a href="#">Nonhlanhla</a>
+        <nav><a href="#"><i class="fas fa-bars"></i></a></nav>
+        
```

Comment:
- [ ] As requested by the previous reviewer kindly style the header following the style on the Figma design
Your design
<img width="364" alt="Screen Shot 2022-05-27 at 11 17 31" src="https://user-images.githubusercontent.com/28307960/170670358-f7380b1d-84e7-482a-bd89-7f9adf299ec6.png">
Figma design

<img width="364" alt="Screen Shot 2022-05-27 at 11 17 31" src="https://user-images.githubusercontent.com/28307960/170670502-60a5df23-1bb8-4464-aa88-f02a8accf605.png">